src/client/internal.ts

```diff
@@ -53,6 +56,7 @@ export enum APIFromClientTopic {
 export enum APIToClientTopic {
   INTENT = 'INTENT',
   CONTEXT = 'CONTEXT',
+  CHANNEL_CONTEXT = 'CHANNEL-CONTEXT'
```

Comment:
Then it becomes inconsistent with `CONTEXT`, and I'd rather keep consistency. Unless we're cool with this being a breaking change, and these become `INTENT_RAISED`, `CONTEXT_SENT`, `CHANNEL_CONTEXT_SENT`?

Reply:
Hmm, best I have for this is `HANDLE_CHANNEL_CONTEXT`. Any narrow verb doesn't work, as this can be sent due to joining a channel, and app explicitly broadcasting a context, or due to an app being opened with a context. And more generic verbs like 'sent' are implicit just from the fact we're setting up an API